platform/security/sts/security-sts-server/src/main/java/ddf/security/sts/SubjectDNConstraintsInterceptor.java

```diff
@@ -74,8 +75,8 @@ public void handleMessage(Message message) throws Fault {
 
     public Collection<Pattern> setSubjectDNPatterns(String subjectDNConstraints) {
```

Comment:
✏️  The name of this method is somewhat confusing. I was expecting it to set something on the object but it really just converts the `subjectDNConstraints` and returns a list of `Pattern`. Should we consider renaming it?